Lib/diffenator/glyphs.py

```diff
@@ -0,0 +1,22 @@
+"""Dump a fontfs glyf table"""
+from diffenator.font import InputFont
+from fontTools.pens.areaPen import AreaPen
+
+
+def glyph_area(glyphset, glyph):
+    """Get the surface area of a glyph"""
+    pen = AreaPen(glyphset)
+    glyphset[glyph].draw(pen)
+    return pen.value
+
+
+def dump_glyphs(font):
+    table = [r for r in font.input_map.values()]
+    glyphset = font.getGlyphSet()
+
+    new_table = []
+    while table:
+        r = table.pop(0)
+        new_r = {'glyph': r, 'area': glyph_area(glyphset, r.name)}
+        new_table.append(new_r)
+    return new_table
```

Comment:
why the while loop here? A for loop would be better.
And make sure you sort when you construct a list while iterating over a dict (or any unsorted collection), otherwise you get different orderings at every run (unsorted collections use a random hash seed in python3). Unless in this particular case you don't care about the order of the items in the returned list.

```python
glyphset = font.getGlyphSet()
return [{'glyph': glyph, 'area': glyph_area(glyphset, name)}
        for name, glyph in sorted(font.input_map.items())]
```

Reply:
Thanks for this. Was most likely written at 1am.